### PR TITLE
BE: MozFest Spaces Cards

### PR DESCRIPTION
**Pull request by b-ggs**
Related PRs/issues #7428

Created a new StructBlock field called `SpaceCardListBlock` containing a heading and a list of child blocks named `SpaceCardBlock` that have their own titles, plain-text bodies, images, and links to either a Wagtail page or an external URL.

Also added a card template similar to an existing card block.

Screen Shot 2021-09-24 at 7 00 57 PM

## Checklist

**Changes in Models:**
- [X] Did I squash my migration?

### CODE REVIEW

**Comment by b-ggs** [comment]
Hey @SharmaineLim and @Pomax! Whenever it's convenient, would you mind reviewing this new StructBlock field? Thank you!

File: network-api/networkapi/wagtailpages/pagemodels/customblocks/space_card_list_block.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        help_text='Body text of the card.'
+    )
+
+    link = blocks.StreamBlock(
+        [
+            ('internal', blocks.PageChooserBlock(help_text='Page that this card should link out to.')),
+            ('external', blocks.URLBlock(help_text='URL that this card should link out to.')),
+        ],
+        max_num=1,
+    )
```
Comment (SharmaineLim) [inline network-api/networkapi/wagtailpages/pagemodels/customblocks/space_card_list_block.py:35]: Also, something to check: do pages here already have `listing_image`, `listing_title` and `listing_summary`?

**Review by SharmaineLim — APPROVED** [review]
Overall, LGTM. See comment.

File: network-api/networkapi/wagtailpages/pagemodels/customblocks/space_card_list_block.py (hunk 1 above)
Comment (b-ggs) [reply network-api/networkapi/wagtailpages/pagemodels/customblocks/space_card_list_block.py:35]: This new StructBlock shares the most similarity to [`CardGrid`](https://github.com/mozilla/foundation.mozilla.org/blob/main/network-api/networkapi/wagtailpages/pagemodels/customblocks/card_grid.py#L34).

**Pull request merged**